cmd/broadcaster-cli/app/utxos/broadcast/broadcast.go

```diff
@@ -114,7 +113,7 @@ var Cmd = &cobra.Command{
 			return fmt.Errorf("failed to create client: %v", err)
 		}
 
-		wocClient := woc_client.New(!isTestnet, woc_client.WithAuth(wocApiKey), woc_client.WithLogger(logger))
+		wocClient := wocclient.New(!isTestnet, wocclient.WithAuth(wocAPIKey), wocclient.WithLogger(logger))
```

Comment:
I think `woc_client` could remain as it is, not?

Reply:
Done